<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1306</width>
    <height>827</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #202429; /* Fallback for old browsers */
/*background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #141212, stop:0.48 #0f2d29, stop:1 #262626); */
color:white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: #202429; /* Fallback for old browsers */
/*background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #141212, stop:0.48 #0f2d29, stop:1 #262626); /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
</string>
   </property>
   <widget class="QLineEdit" name="LineEdit">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>730</y>
      <width>561</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background: #2c3137;
    color: #000000;
	border:none;
	border-radius:23 px;

}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>730</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000;
color: #fff;
border: red 2px solid;
border-radius: 80px</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="text">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : #FFF</string>
    </property>
    <property name="locale">
     <locale language="Filipino" country="Philippines"/>
    </property>
    <property name="text">
     <string>IMAGES</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>110</y>
      <width>301</width>
      <height>341</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Display</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #2a2e32;
border:none;

</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>301</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>70</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #fff;
color: #000000;

</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContents</enum>
    </property>
   </widget>
   <widget class="QWidget" name="history_container" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>271</width>
      <height>831</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:30%;</string>
    </property>
    <widget class="QScrollArea" name="History">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>271</width>
       <height>831</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px; background: #2a2e32
 </string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>271</width>
        <height>831</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Georgia</family>
         <pointsize>21</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #FFF
</string>
       </property>
       <property name="text">
        <string>SEARCH HISTORY</string>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>251</width>
         <height>611</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="read">
        <property name="spacing">
         <number>8</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>70</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #fff;
color: #000000;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="data">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFF</string>
    </property>
    <property name="text">
     <string>TEXT</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="gallery">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>601</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #2a2e32;

border-radius:30px;
</string>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>601</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>470</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Eras Bold ITC</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#565869</string>
    </property>
    <property name="text">
     <string>WEBOASIS</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1306</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWEBOASIS">
    <property name="title">
     <string>WEBOASIS</string>
    </property>
    <addaction name="actionHISTORY"/>
    <addaction name="actionCUt"/>
    <addaction name="actionCOPY"/>
   </widget>
   <addaction name="menuWEBOASIS"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFILE">
   <property name="text">
    <string>FILE</string>
   </property>
  </action>
  <action name="actionHISTORY">
   <property name="text">
    <string>FILE </string>
   </property>
  </action>
  <action name="actionCUt">
   <property name="text">
    <string>CUT</string>
   </property>
  </action>
  <action name="actionCOPY">
   <property name="text">
    <string>COPY</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
